.pullapprove.yml

```diff
@@ -48,8 +48,7 @@ groups:
           - "*.md"
         exclude:
           - "vendor/*"
-          - "cmd/fetchbranches/vendor/*"
-          - "cmd/localCI/vendor/*"
+          - "*/vendor/*"
```

Comment:
@jodh-intel not sure I quite understand - the two vendor excludes are next to each other, and I see no other vendor excludes in this file?
https://github.com/grahamwhaley/tests/blob/15c6854485ccdb35156f131cd8d9b850812112bb/.pullapprove.yml#L50
Am I mis-understanding, or perhaps did you mis-read the patch?

Just went to look at pullapprove yaml comments - yep, the example on their front page has a number in it:
http://docs.pullapprove.com/
and, in fact, our file here has one:
https://github.com/grahamwhaley/tests/blob/15c6854485ccdb35156f131cd8d9b850812112bb/.pullapprove.yml#L7

So, sure, let me go add a couple of comments around the vendor exclusion bits...